action.yml

```
@@ -1,11 +1,17 @@
 name: Gitolize
 description: Wrap a shell command with gitolize
 inputs:
+  ansi:
```

What about a bit longer input name that might be more readable in places this action is used and the its description is not available? (or something shorter than proposed but still hinting what the context is)
```suggestion
  keep-ansi-in-commit-message:
```

---

Done in 583316659c366007d104ef7abf21adb1ead1ef71.